.quickstart/quickstart.yml

```diff
@@ -3,6 +3,12 @@ schema_key: reddit_ads_schema
 
 dbt_versions: ">=1.3.0 <2.0.0"
 
+table_variables:
+  reddit_ads_campaign_country_report_enabled:
+    - campaign_country_report
+  reddit_ads_campaign_country_conversions_report_enabled:
+    - campaign_country_conversions_report
```

Comment:
Same note from source to rename variables

Reply:
updated